grpcbp/prometheus.go

```diff
@@ -55,6 +55,7 @@ var (
 	serverActiveRequestsLabels = []string{
 		serviceLabel,
 		methodLabel,
+		typeLabel,
```

Comment:
FWIW I don't actually see the `type` label in the spec for `grpc_server_active_requests` and `grpc_client_active_requests`, we should add it? (This does appear to match v2 though)